ui/packages/atlasmap-core/src/services/initialization.service.ts

```diff
@@ -650,7 +657,11 @@ ${error.status} ${error.statusText}`,
     inspectionParameters?: { [key: string]: string }
   ): DocumentDefinition {
     const model: DocumentInitializationModel = new DocumentInitializationModel();
-    model.id = name;
+    model.name = name;
+    model.id =
```

Comment:
I'd assing a GUID to allow mutliple same-type&same-name documents.

Reply:
Are we sure we want to allow multiple documents with the exact same name to be imported into the panels?  The user wanted `address.json` and `address.xml` to be allowed.  I don't see a demand for 100 `address.json` documents and adding a global unique ID will add to the complexity.

Reply:
Yes that has to be possible. For example retrieving address object twice but with a different condition, where schema is same.